core/build.sbt

```diff
@@ -2,9 +2,9 @@ import Dependencies._
 
 name := "pureconfig"
 
-organization := "com.github.melrief"
+organization := "com.github.pureconfig"
```

Comment:
In order to be accepted for publishing at Sonatype, the organization name has to follow a DNS-like name and you have to have some "degree" of control over it. I personally prefer that way too, both with Maven dependencies and Java packages - it is a good measure to ensure that the names are unique.